main.cpp

```diff
@@ -131,12 +144,13 @@ void Cat::scratchFurniture(bool isChair, int numFurnitureDestroyed)
     }
         
 }
-void Cat::makeNoise(std::string noise, bool foodBowlEmpty)
+void Cat::makeNoise(bool foodBowlEmpty)
 {
+    std::string noise = "";
     if(foodBowlEmpty)
-    {
         noise = "meow";
-    }
+    noise = "purr";
+    std::cout << "Cat goes: " << noise << std::endl;
 }
```

Comment:
Restore the curly braces.
Based on the logic you have written, `noise` is always `purr`.  
use `if/else` to correct the logic.
- [ ] revise